src/sentry/static/sentry/app/views/organizationDashboard/platformList.jsx

```diff
@@ -18,8 +19,8 @@ class PlatformList extends React.Component {
 
   getIcon(platform) {
     return (
-      <StyledPlatformIconWrapper key={platform} className={platform}>
-        <StyledPlatformIcon className={`platformicon platformicon-${platform}`} />
+      <StyledPlatformIconWrapper key={platform}>
+        <StyledPlatformIcon platform={platform} size="24" className={platform} />
```

Comment:
Do we still need className here?

Reply:
Probably not, but I left it there so the test would still work

Reply:
Let's remove it, we can update the test to search via props, e.g.

```javascript
    expect(platformList.find('StyledPlatformIcon[platform="javascript"]')).toHaveLength(1);
```